src/System.ServiceProcess.ServiceController/src/System/ServiceProcess/ServiceProcessDescriptionAttribute.cs

```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
+// <copyright file="ServiceProcessDescriptionAttribute.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>                                                                
+//------------------------------------------------------------------------------
+
```

Comment:
Thanks for pointing out. I decided to not include this file at all but I forget to remove it from my original commit.